docs/Glossary/index.md

```diff
@@ -0,0 +1,251 @@
+# Glossary
+
+This glossary provides definitions and explanations of commonly used terms, abbreviations, and phrases.
+
+---
+
+## A
+
+### Atomic Move
+
+This is a real file move, carried out instantaneously, rather than by copying a file from one location to another and then deleting the file from the original folder.
+
+---
+
+## B
+
+---
+
+## C
+
+### CF
+
+Depending on its context, whether hosting or media management applications, this can refer to **CloudFlare** or [Custom Formats](#custom-format).
+
+### Cross-Seed
+
+The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org){:target="\_blank" rel="noopener noreferrer"} that automates this process.
+
+`Cross-Seed` and `crossseed` or `xseed` often refer to the application, while `cross seed` or `xseed` can refer to the action.
+
+### Custom Format
+
+!!!info
+
+    Also known as: CF
+
+A feature that uses pattern matching on a release, based on the information implied through the release name on the corresponding indexer during a search. Users can customize patterns to match a Custom Format, and Custom Formats can then be assigned scores individually in each [Quality Profile](#quality-profile).
+
+After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.
```

Comment:
Doesn't this depend on the naming scheme the user is using? If they're not using the relevant tokens in the naming then those things won't be retained?

Reply:
```suggestion
After the release has been imported, when using the TRaSH recommended naming scheme, the Custom Format score will automatically be updated to match the properties of the _actual_ media file, rather than strictly by its original naming alone.

Reply:
Beautiful.